tests/runtime/probe

```diff
@@ -23,21 +25,25 @@ NAME kprobe_offset
 RUN bpftrace -v -e 'kprobe:vfs_read+0 { printf("SUCCESS '$test' %d\n", pid); exit(); }'
 EXPECT SUCCESS kprobe_offset [0-9][0-9]*
 TIMEOUT 5
+AFTER ./testprogs/syscall read
 
 NAME kprobe_offset_fail_size
 RUN bpftrace -v -e 'kprobe:vfs_read+1000000 { printf("SUCCESS '$test' %d\n", pid); exit(); }'
 EXPECT Offset outside the function bounds \('vfs_read' size is*
 TIMEOUT 5
+AFTER ./testprogs/syscall read
```

Comment:
This bpftrace invocation should fail before it even loads so I don't think we need it here

Reply:
Yeah, I'd agree.